docs/documentation/development/bindings/discovery-services.md

```diff
@@ -114,7 +120,14 @@ Normally, older discovery results already in the inbox are left untouched by a n
 
 ### Compare with existing results
 
-In some cases it might be of interest for a discovery service if the same device was discovered before or even a Thing already exists. Therefore, a discovery service may implement the ExtendedDiscoveryService interface and use the injected callback in order to access the DiscoveryResult or Thing for a given ThingUID, if it exists. 
+In some cases it might be of interest for a discovery service if the same device was discovered before or even a Thing already exists. Therefore, a discovery service may implement the ExtendedDiscoveryService interface and use the injected callback in order to access the DiscoveryResult or Thing for a given ThingUID, if it exists.
```

Comment:
If you want to make it consistent, why did you write "Thing" with a capital "T" here? :) Also please add a line break after each sentence.

Reply:
Oh, you are right. Thanks.